stdlib/2.7/typing.pyi

```diff
@@ -161,7 +161,7 @@ class Mapping(Sized, Iterable[_KT], Container[_KT], Generic[_KT, _VT]):
     @abstractmethod
     def __getitem__(self, k: _KT) -> _VT: ...
     # Mixin methods
-    def get(self, k: _KT, default: _VT = ...) -> _VT: ...
+    def get(self, k: _KT, default: _T = ...) -> Union[_VT, _T]: ...
```

Comment:
For get() the default needs to be equal to None (though maybe an overload would still be clearer).